### PR TITLE
Change parse-block to emit actual block object, not just number

### PR DESCRIPTION
**Pull request by epheph**
/cc @MicahZoltu for type sanity

### CODE REVIEW

File: index.d.ts [hunk 1]
```diff
@@ -1,5 +1,8 @@
 /// <reference types="node" />
```
Comment (MicahZoltu) [inline index.d.ts]: Unrelated to this PR, but this comment is the old way of doing this and shouldn't be included anymore I believe.

Comment (epheph) [reply index.d.ts]: Want me to yank while i'm in there?

Comment (MicahZoltu) [reply index.d.ts]: Totally your call.  I could be wrong, so you should set aside some time to make sure yanking it doesn't break anything if you do.  :)

**Comment by epheph** [comment]
Coverage is flagging this PR because i reduced code in a covered function?

**Comment by MicahZoltu** [comment]
😆 🤣

**Pull request merged**